### DISCUSSION TITLE
`kitty +kitten ssh <remote>` fails under tmux, post-0.25.x

### DISCUSSION THREAD

**Discussion by megalithic** [kovidgoyal/kitty#5240] [Q&A]
Good day!

**DISCLAIMER:**
> I _KNOW_ tmux is a touchy subject here, and I'm working to migrate away from it, but I'm not there yet. I'd love to just use kitty, and only kitty for all the things I used with tmux... one day.

Before version 0.25.x, I was able to connect to various remote servers with `kitty +kitten ssh <remote>` within a tmux session..

Now, however, it hangs; here's verbose output: https://gist.github.com/megalithic/8233c7848e312ec1b13f345da39875d1#file-initial-connection-attempt

I've tried running with an empty `kitty.conf` _AND_ with an empty `tmux.conf`, to ensure my settings aren't/weren't messing with new things, it still hangs either way. Also, I've tried with and without shell_integration enabled (and manual shell_integration setup for `zsh` -- the shell i use).

- [`kitty.conf`](https://github.com/megalithic/dotfiles/blob/main/config/kitty/kitty.conf)
- [`tmux.conf`](https://github.com/megalithic/dotfiles/blob/main/config/tmux/tmux.conf)

It _DOES_ connect outside of tmux, though.

Any thoughts/suggestions? Anyone else hit this?

**Comment by page-down** [comment]
There is a recent GitHub issue that discusses this.
Unfortunately the new ssh kitten cannot be used in tmux.

https://github.com/kovidgoyal/kitty/issues/5227

**Reply by megalithic** [reply]
Thanks @page-down :/ I was afraid of that.
